packs/octopusdeploy/actions/get_deployments.py

```diff
@@ -0,0 +1,14 @@
+from lib.actions import OctopusDeployAction
+
+__all__ = [
+    'GetDeploymentsAction'
+]
+
+
+class GetDeploymentsAction(OctopusDeployAction):
+    def run(self, project_id):
+        result = self.make_get_request(action="deployments",
+                                       params={
+                                           'projects': project_id
+                                       })
+        return result.get('Items', None)
```

Comment:
It looks like this API call returns a list. If so it might be better to return empty list instead if the `Items` attribute is not available.

This way it's easier to use it inside the actions since you don't need to check for None.